src/components/pages/HomePage/HomePage.js

```diff
@@ -30,6 +31,12 @@ const HomePage = ({ data: { cms } }) => {
 
       <FAB className={styles.fab} />
 
+      <SectionHeadline
+        className={styles.sectionHeadline}
+        preHeadline={'como'}
+        headline={'Fazemos'}
```

Comment:
It's missing translations, isn't it?

```suggestion
        preHeadline={t('how')}
        headline={t('we_do')}
```

Reply:
Yes. But again, it was not my focus. I prefer to build the components first and wire up the data later when I'm finishing up the `Page` layout.